app/models/vote.rb

```diff
@@ -90,6 +90,7 @@ def associate_previous_vote
   end
 
   def age_previous_votes
+    Vote.connection.execute("SET CONSTRAINTS vote_age_per_user_per_motion DEFERRED")
```

Comment:
No worries! There are some gems that make this more seamless but sadly they're either out of date (https://github.com/pedz/activerecord_constraints) or only support deferring foreign key constraints (https://github.com/zoocasa/activerecord-postgresql-extensions).